provider/testdata/Cargo.toml

```diff
@@ -299,66 +301,43 @@ displaydoc = { version = "0.2.3", default-features = false, optional = true }
-icu_list = { version = "1.0.0-beta1", path = "../../components/list", optional = true }
-icu_locid_transform = { version = "1.0.0-beta1", path = "../../components/locid_transform", optional = true }
-icu_normalizer = { version = "1.0.0-beta1", path = "../../components/normalizer", optional = true }
-icu_plurals = { version = "1.0.0-beta1", path = "../../components/plurals", optional = true }
-icu_properties = { version = "1.0.0-beta1", path = "../../components/properties", optional = true }
-icu_segmenter = { version = "1.0.0-alpha1", path = "../../experimental/segmenter", optional = true, features = ["lstm"] }
-icu_timezone = { version = "1.0.0-beta1", path = "../../components/timezone", optional = true }
-icu_collections = { version = "1.0.0-beta1", path = "../../components/collections", optional = true }
-litemap = { version = "0.5", path = "../../utils/litemap", optional = true, default-features = false }
-tinystr = { version = "0.6", path = "../../utils/tinystr", optional = true }
-zerovec = { version = "0.8", path = "../../utils/zerovec", optional = true }
+icu_locid = { version = "1.0.0-beta1", path = "../../components/locid" }
+icu_collections = { version = "1.0.0-beta1", path = "../../components/collections" }
+litemap = { version = "0.5", path = "../../utils/litemap", default-features = false }
+tinystr = { version = "0.6", path = "../../utils/tinystr" }
+zerovec = { version = "0.8", path = "../../utils/zerovec" }
 
-[dev-dependencies]
-icu_plurals = { version = "1.0.0-beta1", path = "../../components/plurals" }
+# databake options
+icu_calendar = { version = "1.0.0-beta1", path = "../../components/calendar", default-features = false, optional = true }
+icu_casemapping = { version = "0.2", path = "../../experimental/casemapping", default-features = false, optional = true }
+icu_collator = { version = "1.0.0-beta1", path = "../../components/collator", default-features = false, optional = true }
+icu_datetime = { version = "1.0.0-beta1", path = "../../components/datetime", default-features = false, optional = true }
+icu_decimal = { version = "1.0.0-beta1", path = "../../components/decimal", default-features = false, optional = true }
+icu_list = { version = "1.0.0-beta1", path = "../../components/list", default-features = false, optional = true }
+icu_locid_transform = { version = "1.0.0-beta1", path = "../../components/locid_transform", default-features = false, optional = true }
+icu_normalizer = { version = "1.0.0-beta1", path = "../../components/normalizer", default-features = false, optional = true }
+icu_plurals = { version = "1.0.0-beta1", path = "../../components/plurals", default-features = false, optional = true }
+icu_properties = { version = "1.0.0-beta1", path = "../../components/properties", default-features = false, optional = true }
+icu_segmenter = { version = "1.0.0-alpha1", path = "../../experimental/segmenter", features = ["lstm"], default-features = false, optional = true }
```

Comment:
Thought (optional): maybe you should make `segmenter_lstm` another optional feature like you did with `datetime_experimental`

Reply:
Segmenter itself is still experimental though